server/Opendatacam.js

```diff
@@ -612,34 +612,30 @@ module.exports = {
   },
 
   sendUpdateToClient: function() {
```

Comment:
Should we rename this function to reflect that this sends out all the updates and not just to a single client?

Reply:
I've renamed to `sendUpdateToClients`.